packages/panzer/disc-fe/src/Panzer_Workset.cpp

```diff
@@ -88,10 +88,11 @@ WorksetDetails::setup(const panzer::LocalMeshPartition & partition,
   fc->setup(partition);
   _face_connectivity = fc;
 
-  setupNeeds(partition.cell_topology, partition.cell_vertices, needs);
+  setupNeeds(partition, partition.cell_topology, partition.cell_vertices, needs);
 }
 
-void WorksetDetails::setupNeeds(Teuchos::RCP<const shards::CellTopology> cell_topology,
+void WorksetDetails::setupNeeds(const panzer::LocalMeshPartition &mesh_info,
```

Comment:
We're going to need to do this without passing the mesh partition into the setupNeeds call. I believe this call is used by a lot of other codes, and other codes do not generate LocalMeshInfo objects.

The information you are using in the LocalMeshInfo object already exists in the Workset in the SubcellConnectivity object.

Reply:
@seamill, thanks for the note.  I've revised the PR accordingly.  I also fixed the failing test (check_rotation_matrices was enforcing the old, incorrect behavior on virtual cells).